src/images/socialMedia/insta.svg

```diff
@@ -0,0 +1,14 @@
+<path d="M98.0217 73.3835C84.6871 73.3835 73.868 83.9289 73.868 96.9277C73.868 109.926 84.6871 120.472 98.0217 120.472C111.357 120.472 122.175 109.926 122.175 96.9277C122.175 83.9289 111.357 73.3835 98.0217 73.3835ZM98.0217 112.2C89.3709 112.2 82.3538 105.361 82.3538 96.9277C82.3538 88.4945 89.3709 81.6552 98.0217 81.6552C106.673 81.6552 113.69 88.4945 113.69 96.9277C113.69 105.361 106.673 112.2 98.0217 112.2Z" fill="current"/>
+<path d="M128.77 72.4535C128.77 75.4889 126.246 77.95 123.131 77.95C120.017 77.95 117.492 75.4889 117.492 72.4535C117.492 69.4175 120.017 66.957 123.131 66.957C126.246 66.957 128.77 69.4175 128.77 72.4535Z" fill="current"/>
+</g>
+<defs>
+<filter id="filter0_d" x="0" y="0.0927734" width="196.007" height="193.652" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
```

Comment:
Just a note - these ids (e.g.: `filter0_d`, or `paint0_linear` when we have linear gradients) in the SVG are global, so we might need to rename them if it conflicts with others. I don't see a problem here so we don't need to change them, but if the color / filter looks off in the future, that may be why.

Reply:
That's good to note, we'll definitely track of these throughout the future.